projects/rero/ng-core/src/lib/record/editor/widgets/label/label.component.ts

```diff
@@ -44,6 +44,9 @@ export class LabelComponent {
    * @returns boolean, true if the menu should be displayed
    */
   hasMenu(field: FormlyFieldConfig) {
+    if (field.fieldGroup && field.fieldGroup.length > 0 && field.fieldGroup[0].type === 'multischema') {
+      return false;
+    }
```

Comment:
for my information : in which case this is happening ?

Reply:
Allows to deactivate the arrow on the parent, because it produced a duplicate (Ex: identifier on the 2 elements)

Before:
<img width="510" alt="before" src="https://user-images.githubusercontent.com/48578/150733154-66c2f604-2ca6-49c7-810f-eb10bef05a49.png">

After:
<img width="418" alt="after" src="https://user-images.githubusercontent.com/48578/150733194-6530d2b0-8fbf-4168-bef3-1377f4724177.png">